Formula/librttopo.rb

```diff
@@ -37,6 +43,6 @@ def install
       }
     EOS
     system ENV.cc, "test.c", "-I#{include}", "-L#{lib}", "-lrttopo", "-o", "test"
-    assert_equal version.to_s, shell_output("./test")
+    assert_equal stable.version.to_s, shell_output("./test")
```

Comment:
interesting, what is the difference between the two (`version` vs `stable.version`)?

Reply:
I see, this is for the `head` build test. But I doubt folks would run `brew test` for the `head` build.